tests/src/PrivateClass.php

```diff
@@ -0,0 +1,8 @@
+<?php
```

Comment:
Can you move this class/interface into the test file `namespaceUsagesExcludeAttribute.php`? I think it should work and maybe it will be more clear how the feature works.

Reply:
Moved over... it breaks composers autoloading when using `class_exists` or `interface_exists` - but those checks aren't required since we're using PHPStan's reflector which is much more robust - I've still added a try / catch just in case the Reflector can't find the class which mimics the original behaviour.